modules/setting/setting.go

```diff
@@ -1277,15 +1272,30 @@ func NewXORMLogService(disableConsole bool) {
 	}
 }
 
+// Cache represents cache settings
+type Cache struct {
+	Adapter  string
+	Interval int
+	Conn     string
+}
+
+var (
+	// CacheService the global cache
+	CacheService *Cache
+)
+
 func newCacheService() {
-	CacheAdapter = Cfg.Section("cache").Key("ADAPTER").In("memory", []string{"memory", "redis", "memcache"})
-	switch CacheAdapter {
+	sec := Cfg.Section("mailer")
```

Comment:
Why `mailer` not `cache`?

Reply:
Ouch, good catch 👍

Reply:
@lunny do you mean disabling all caching or only commit count caching?

Reply:
I mean all the cache. If I have only serval repos on my private VPS. I maybe don't want to enable cache to spent the memory.